02_programs/02a_create_analysis_file.Rmd

```diff
@@ -130,82 +124,50 @@ stopifnot(eeptools::isid(cp_oz2, c("GEOID_b", "post", "quarter")))
 # Winsorize
 ar_oz2 <- ar_oz2 %>%
   mutate(
-    violent_ellen_winsorize_ar = Winsorize(violent_ellen, probs = c(0.01, 0.99)),
-    property_ellen_winsorize_ar = Winsorize(property_ellen, probs = c(0.01, 0.99))
-  ) %>%
-  pivot_wider(
-    id_cols = c("GEOID_b", "GEOID", "post", "distance_boundary"), names_from = "quarter", values_from =
-      ends_with("_ar")
+    violent_ellen_winsorize_ar = Winsorize(violent_ellen_ar, probs = c(0.01, 0.99)),
+    property_ellen_winsorize_ar = Winsorize(property_ellen_ar, probs = c(0.01, 0.99))
   ) %>%
-  left_join(acs_oz_vars, by = "GEOID") %>%
-  mutate(across(starts_with("violent_ellen"), ~ if_else(is.na(.), 0, .))) %>%
-  mutate(across(starts_with("property_ellen"), ~ if_else(is.na(.), 0, .)))
+  mutate(across(starts_with("violent_ellen_ar"), ~ if_else(is.na(.), 0, .))) %>%
+  mutate(across(starts_with("property_ellen_ar"), ~ if_else(is.na(.), 0, .)))
 
 cp_oz2 <- cp_oz2 %>%
   mutate(
-    violent_ellen_winsorize_cp = Winsorize(violent_ellen, probs = c(0.01, 0.99)),
-    property_ellen_winsorize_cp = Winsorize(property_ellen, probs = c(0.01, 0.99))
+    violent_ellen_winsorize_cp = Winsorize(violent_ellen_cp, probs = c(0.01, 0.99)),
+    property_ellen_winsorize_cp = Winsorize(property_ellen_cp, probs = c(0.01, 0.99))
   ) %>%
-  pivot_wider(
-    id_cols = c("GEOID_b", "GEOID", "post", "distance_boundary"),
-    names_from = "quarter", values_from = ends_with("_cp")
-  ) %>%
-  left_join(acs_oz_vars, by = "GEOID") %>%
```

Comment:
I added this left join here because we need to remerge the acs data. If there are no crime incidents in the data, these columns will be missing in the analysis file. It is better to merge them here rather than aggregate them.